backends/test/suite/flow.py

```diff
@@ -21,42 +22,35 @@ class TestFlow:
 
     backend: str
     """ The name of the target backend. """
-
-    tester_factory: Callable[[], Tester]
+    
+    tester_factory: Callable[..., Tester]
     """ A factory function that returns a Tester instance for this lowering flow. """
 
+    quantize: bool = field(default=False)
+    """ Whether to tester should run the quantize stage on the model. """
+    
+    quantize_stage_factory: Callable[..., Quantize] | None = None
+    """ A factory function which instantiates a Quantize stage. Can be None to use the tester's default. """
```

Comment:
why an extra flag?

Reply:
The specific reason is that if quantize_stage_factory isn't provided, it will use the default Quantize stage from the tester. I could maybe just always require the caller to provide quantize_stage_factory.